### PR TITLE
[nibiru] Add liquid staked NIBI for Nibiru

### PR DESCRIPTION
**Pull request by Unique-Divine**
- Closes https://github.com/NibiruChain/nibiru/issues/2067

### CODE REVIEW

File: nibiru/assetlist.json [hunk 1]
```diff
@@ -25,7 +25,7 @@
       ],
       "base": "unibi",
       "name": "Nibiru",
-      "display": "nibi",
+      "display": "NIBI",
```
Comment (JeremyParish69) [inline nibiru/assetlist.json:28]: This unit is not defined and will not pass tests. It was fine before, and I assume you mistake what the 'display' property means. "Symbol" is shown to users, while "display" is only used to determine the number decimal points for the asset amounts.

File: nibiru/assetlist.json [hunk 2]
```diff
@@ -43,6 +43,42 @@
...
+        },
+        {
+          "denom": "stNIBI",
+          "exponent": 6
+        }
+      ],
+      "base": "tf/nibi1udqqx30cw8nwjxtl4l28ym9hhrp933zlq8dqxfjzcdhvl8y24zcqpzmh8m/ampNIBI",
+      "name": "Liquid Staked Nibiru (Eris)",
+      "display": "stNIBI",
+      "symbol": "stNIBI",
```
Comment (JeremyParish69) [inline nibiru/assetlist.json:66]: Please define the trace for this, showing how it relates to NIBI (using trace type: liquid-stake)

Comment (Unique-Divine) [reply nibiru/assetlist.json:66]: Updated in 5f8a4e8ee8cfceca0a8d914b5623e3876f0175ff

File: nibiru/assetlist.json (hunk 1 above)
Comment (Unique-Divine) [reply nibiru/assetlist.json]: I see, so the display has to match one of the denom units, while symbol is the actual string that people see in most wallets

Fixed in 5f8a4e8ee8cfceca0a8d914b5623e3876f0175ff

**Pull request merged**